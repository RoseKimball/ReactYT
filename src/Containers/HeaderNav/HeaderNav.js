import React, { Component } from 'react';
import Logo from '../../Assets/Images/Logo.png';
import { Image, Menu, Input, Form, Icon } from 'semantic-ui-react';
import './HeaderNav.scss';

export class HeaderNav extends Component {
    render() {
        return (
            <Menu borderless className='top-menu' fixed='top'>
                <Menu.Item header className='logo'>
                    <Image src={Logo} size='tiny'/>
                </Menu.Item>
                <Menu.Menu className='nav-container'>
                    <Menu.Item className='search-input'>
                        <Form>
                            <Form.Field>
                                <Input 
                                placeholder='Search...'
                                size='small'
                                action='Go'
                                />
                            </Form.Field>
                        </Form>
                    </Menu.Item>
                    <Menu.Menu position='right'>
                        <Menu.Item>
                            <Icon className='header-icon' name='video camera' size='large'></Icon>
                        </Menu.Item>
                        <Menu.Item>
                            <Icon className='header-icon' name='grid layout' size='large'></Icon>
                        </Menu.Item>
                        <Menu.Item>
                            <Icon className='header-icon' name='chat' size='large'></Icon>
                        </Menu.Item>
                        <Menu.Item>
                            <Icon className='header-icon' name='alarm' size='large'></Icon>
                        </Menu.Item>
                        <Menu.Item name='avatar'>
                            <Image src='http://via.placeholder.com/80x80'/>
                        </Menu.Item>
                    </Menu.Menu>
                </Menu.Menu>
            </Menu>
        );
    };
};

export default HeaderNav;