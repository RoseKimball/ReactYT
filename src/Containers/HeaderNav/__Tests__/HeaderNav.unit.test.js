import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, mount, render } from 'enzyme';
import HeaderNav from '../HeaderNav';

describe('HeaderNav', () => {
    test('renders HeaderNav', () => {
        const wrapper = shallow(
            <HeaderNav />
        );
        expect(wrapper).toMatchSnapshot();
    });

})
