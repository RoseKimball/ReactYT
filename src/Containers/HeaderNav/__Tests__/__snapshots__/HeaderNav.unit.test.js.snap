// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderNav renders HeaderNav 1`] = `ShallowWrapper {}`;

exports[`renders HeaderNav 1`] = `ShallowWrapper {}`;
